backend/src/main/java/com/example/dontwastetomuch/game/GameController.java

```diff
@@ -31,8 +34,9 @@ public Game getOneGame(@PathVariable String gameId){
         return gameService.getOneGame(gameId);
     }
 
-    @PutMapping()
-    public void editGame(@RequestBody Game game){
+    @PutMapping("/{gameId}")
+    public void editGame(@PathVariable String gameId){
+        Game game = getOneGame(gameId);
```

Comment:
The game is not changes. It is retrieved from the database and directly saved again.

Reply:
I do it in my GameService, under line 39 i do gameService.editGame(game);
And in my service i have a method edit game. In this method the approved status get changed.

    public void editGame(Game game) {
        if (game.isApproved()) {
            game.setApproved(false);
            gameRepo.save(game);
        }else {
            game.setApproved(true);
            gameRepo.save(game);
        }
    }